src/main/java/teammates/ui/controller/InstructorFeedbackResultsPageData.java

```diff
@@ -918,7 +923,9 @@ private void buildTeamsStatisticsTableForSectionPanel(
                      Set<String> teamsInSection) {
         Map<String, List<InstructorFeedbackResultsQuestionTable>> teamToStatisticsTables = new HashMap<String, List<InstructorFeedbackResultsQuestionTable>>();
         for (String team : teamsInSection) {
-            if (!responsesGroupedByTeam.containsKey(team)) {
+            // skip team if no responses, 
+            // or if the team is an anonymous student's team or an anonymous team, or is "-"
+            if (!responsesGroupedByTeam.containsKey(team) || !bundle.rosterTeamNameMembersTable.containsKey(team)) {
```

Comment:
Should we extract `bundle.rosterTeamNameMembersTable.containsKey(team)` into some `isTeamAnonymous()` method?
Using a named variable is fine too, but if this is going to be used more than once, a method may be better.